src/applications/letters/containers/AddressSection.jsx

```diff
@@ -9,82 +10,80 @@ import {
 
-    const addressContent = (
-      <div className="step-content">
-        <p>Downloaded documents will list your address as:</p>
-
-        <div className="va-profile-wrapper">
-          <InitializeVAPServiceID>
-            <VAPServicePendingTransactionCategory
-              categoryType={TRANSACTION_CATEGORY_TYPES.VAP_ADDRESS}
-            >
-              <AddressField
-                fieldName={FIELD_NAMES.MAILING_ADDRESS}
-                title={FIELD_TITLES[FIELD_NAMES.MAILING_ADDRESS]}
-              />
-            </VAPServicePendingTransactionCategory>
-          </InitializeVAPServiceID>
-        </div>
-        <p>
-          When you download a letter, it will show this address. If this address
-          is incorrect you may want to update it, but your letter will still be
-          valid even with the incorrect address.
-        </p>
-      </div>
-    );
+  const addressContent = (
+    <div className="step-content">
+      <p>Downloaded documents will list your address as:</p>
 
-    let viewLettersButton;
-    if (location.pathname === '/confirm-address') {
-      viewLettersButton = (
-        <div className="step-content">
-          <button
-            onClick={this.navigateToLetterList}
-            className="usa-button-primary view-letters-button"
-            disabled={emptyAddress}
+      <div className="va-profile-wrapper">
+        <InitializeVAPServiceID>
+          <VAPServicePendingTransactionCategory
+            categoryType={TRANSACTION_CATEGORY_TYPES.VAP_ADDRESS}
           >
-            View Letters
-          </button>
-        </div>
-      );
-    }
+            <AddressField
+              fieldName={FIELD_NAMES.MAILING_ADDRESS}
+              title={FIELD_TITLES[FIELD_NAMES.MAILING_ADDRESS]}
+            />
+          </VAPServicePendingTransactionCategory>
+        </InitializeVAPServiceID>
+      </div>
+      <p>
+        When you download a letter, it will show this address. If this address
+        is incorrect you may want to update it, but your letter will still be
+        valid even with the incorrect address.
+      </p>
+    </div>
+  );
 
-    return (
-      <div>
-        <div>
-          <div aria-live="polite" aria-relevant="additions">
-            {emptyAddress ? noAddressBanner : addressContent}
-          </div>
-        </div>
-        {viewLettersButton}
+  let viewLettersButton;
+  if (location.pathname === '/confirm-address') {
+    viewLettersButton = (
+      <div className="step-content">
+        <button
+          onClick={() => navigateToLetterList(router)}
+          className="usa-button-primary view-letters-button"
+          disabled={emptyAddress}
+        >
+          View Letters
+        </button>
       </div>
     );
   }
-}
 
-function mapStateToProps(state) {
-  return {
-    savedAddress: selectVAPContactInfo(state)?.mailingAddress,
-  };
+  return (
+    <>
+      <div aria-live="polite" aria-relevant="additions">
```

Comment:
I'm not certain that we want to be using any `aria` properties here. This is another one to talk to DK about